include/ipmitool/helper.h

```diff
@@ -38,6 +38,12 @@
 #include <stdio.h>
 #include <string.h>
 
+#ifdef __GNUC__
+#  define __UNUSED__(x) __attribute__((__unused__)) x
```

Comment:
Doesn't look like this new macro is used anywhere within this pull request.
Did you forget to replace occurrences of `__attribute__((__unused__))` with it?

Reply:
Oops. I was originally just going to mark the unused parameters as unused, but then I decided to just remove them. I will remove this.